src/components/Header/Header.js

```diff
@@ -52,7 +52,7 @@ export function Header({children, algoliaFilters}) {
   const {toggleColorMode, colorMode} = useColorMode();
   const [tagBg, tagTextColor] = useTagColors();
   return (
-    <Box pos="sticky" top="0" zIndex="1">
+    <Box pos="sticky" top="0" zIndex="10">
```

Comment:
```suggestion
    <Box pos="sticky" top="0" zIndex="2">
```

Would you be able to change this to `zIndex` of `2`? The main content rests at the default `0` and the `aside` (the sidebar) has an index of `1`. `2` is the minimum needed to push the header to the top of the stack.